cmd/chronograf-migrator/dashboard.go

```diff
@@ -376,22 +376,29 @@ func convertQueries(qs []chronograf.DashboardQuery) []influxdb.DashboardQuery {
 	return ds
 }
 
-type dbrpMapper struct {
-}
+type dbrpMapper struct{}
 
-func (m dbrpMapper) FindBy(ctx context.Context, cluster string, db string, rp string) (*influxdb.DBRPMapping, error) {
-	return nil, errors.New("mapping not found")
-}
-func (m dbrpMapper) Find(ctx context.Context, filter influxdb.DBRPMappingFilter) (*influxdb.DBRPMapping, error) {
+// FindBy returns the dbrp mapping for the specified ID.
+func (d dbrpMapper) FindByID(ctx context.Context, orgID influxdb.ID, id influxdb.ID) (*influxdb.DBRPMappingV2, error) {
 	return nil, errors.New("mapping not found")
 }
-func (m dbrpMapper) FindMany(ctx context.Context, filter influxdb.DBRPMappingFilter, opt ...influxdb.FindOptions) ([]*influxdb.DBRPMapping, int, error) {
-	return nil, 0, errors.New("mapping not found")
 
+// FindMany returns a list of dbrp mappings that match filter and the total count of matching dbrp mappings.
+func (d dbrpMapper) FindMany(ctx context.Context, dbrp influxdb.DBRPMappingFilterV2, opts ...influxdb.FindOptions) ([]*influxdb.DBRPMappingV2, int, error) {
+	return nil, 0, errors.New("mapping not found")
 }
```

Comment:
Are these just unimplemented for now?

Reply:
I am not sure about these. The original implementation was unimplemented, so I just kept it consistent.
I assume that the chronograf migrator does not need a DBRP mapping service, but wants something to go in its place other than nil?

Reply:
Yeah, makes sense to follow the original pattern.